tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -33,13 +33,43 @@ async def tx_in_pool(mempool: MempoolManager, tx_id: bytes32) -> bool:
     return True
 
 
+def assert_rpc_response(response, condition_func=None) -> Dict:
+    assert isinstance(response, dict)
+    assert response.get("success")
+    if condition_func:
+        assert condition_func(response)
+    return response
+
+
+async def wait_rpc_state_condition(timeout, coroutine, params, condition_func) -> Dict:
```

Comment:
I want less callbacks.  But, I don't have the alternative tooling in place yet.  Going forward, try to find ways to avoid callbacks.  Python has a lot of tools.